### DISCUSSION TITLE
Customize appearance to match our site...

### DISCUSSION THREAD

**Discussion by inceed** [bullhorn/career-portal#397] [General]
Currently I do not see a way to customize the CSS to match our website look and feel without modifying the actual plugin itself. Whenever you do an update as I see you have this week, this will likely wipeout this customization. Is this something you plan to address in the future? It seems like such an awkward way to have to customize the look.

**Comment by pbearne** [comment]
Hi @inceed 
Yes, you need to customize the plugin/CSS to style the OSCP
As you are using WordPress for your site. We at Matador Jobs (Bullhorn partner) can help as our WordPress Plugin integrates directly into WordPress all look and feel from your theme are automatically applied.
Reach out to me at paul@matadorjobs.com for a demo/chat to see if we can help.

Paul

**Comment by charlesabarnes** [comment]
Heya,

If customizations are in use for the plugin, I suggest disabling auto-update on the plugin since the WordPress update system tends to get rid of all of the changes.  alternative options are being investigated for app theming.
